content/get-started/multisig-accounts.mdx

```diff
@@ -0,0 +1,201 @@
+---
+title: Multisig Accounts
+seo-title: Multisig Accounts
+---
+
+## Create a multisig account
+
+Iron Fish supports multisig accounts using [FROST](https://frost.zfnd.org/frost.html), a threshold signature scheme.
```

Comment:
I would add an introduction that talks about what multisig accounts are about, what problem they solve, and when they should be used. (I can contribute to this later this week)